akka-actor/src/main/scala/akka/actor/UntypedActor.scala

```diff
@@ -92,6 +92,7 @@ package akka.actor
  *  }
  * }}}
  */
+@Deprecated
 @deprecated("Use AbstractActor instead of UntypedActor.", since = "2.5.0")
 abstract class UntypedActor extends Actor {
```

Comment:
small boyscouting... otherwise java people won't notice it is actually deprecated, and it is an java api afterall.

Reply:
It does not do it for IDEs though, so people using Java would not see a warning in the IDEs that the method is deprecated